source/agora/test/Base.d

```diff
@@ -56,7 +56,7 @@ public class TestNetworkManager : NetworkManager
     /// Used by the unittests in order to directly interact with the nodes,
     /// without trying to handshake or do any automatic network discovery.
     /// Also kept here to avoid any eager garbage collection.
-    public TestAPI[PublicKey] apis;
```

Comment:
Where and how ill they be used directly? I mean if you could just describe a use-case?

Reply:
This comment should have been on the right side.

Reply:
I don't understand the question?

Reply:
"Expose something because it will be used". How will it be used?

Reply:
Oh right. That's an artifact.